src/XrdPfc/XrdPfcFSctl.cc

```diff
@@ -131,6 +132,23 @@ int XrdPfcFSctl::FSctl(const int               cmd,
    rc = SFS_ERROR;
   }
 
+  if (!strcmp(xeq, "cached"))
+  {
+     const char* path = args.ArgP[0];
+     int rval = myCache.LocalFilePath(path, nullptr, 0, XrdOucCache::LFP_Reason::ForInfo);
```

Comment:
So, if I understand this correctly, if at least one block of the file is cached we will get a zero return code and say it's OK.  Is that correct or am I missing something here?

I say that because a previous comment said only if the file is "fully cached". That's the opposite extreme and I do wonder about tat choice as well.

Reply:
At first @bbockelm  was thinking that a file is considered cached if any part of the file is cached. Last week at the Pelican meeting the decision was that the file must be fully cached.

I can’t tell which of the two options would work best.
Perhaps this can be a configuration option.

Reply:
Indeed, this is going to go back and forth until someone figures out what really works in practice. Yes, a configuration option would be best here. I'd go with something like pfc.ifcache <num>% where <num> is a positive integer represent9ing that you return true should the file be at least that percentage in the cache. The question is how to actually get that information to make that decision.